src/components/input/input-short/InputShortCSS.js

```diff
@@ -0,0 +1,44 @@
+import { colors } from "../../colors";
+
+export const InputShortComponentCSS = `
+position: relative;
+  background-color: transparent;
+  width: 474px;
+  height: 22px;
+  border-radius: 8px;
+  border: 2px solid white;
+  gap: 10px;
+  color: white;
+  outline: none;
+  z-index: 0;
+
+  padding: 13px;
+  font-family: "HK Grotesk", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
```

Comment:
Small nit: should `font-size` be in `rem`?

Reply:
Otherwise, everything else lgtm

Reply:
made the changes! make sure to click Approve for the PR